cron.php

```diff
@@ -0,0 +1,464 @@
+<?php
+/*
+ Plugin Name: WP-Cron Control Revisited
+ Plugin URI:
+ Description: Take control of wp-cron execution.
+ Author: Erick Hitter, Automattic
+ Version: 0.1
+ Text Domain: wp-cron-control-revisited
+ */
+
+// Enable in production only for specific sites
+// Otherwise, class WP-Cron Control is loaded
+$whitelisted_sites = array();
+if ( VIP_GO_ENV && ! in_array( FILES_CLIENT_SITE_ID, $whitelisted_sites ) ) {
```

Comment:
We check and define `VIP_GO_ENV` in `000-init.php` (https://github.com/Automattic/vip-go-mu-plugins/blob/cc326452d8745063ffef8032ab46de9b6ef7cc8e/000-vip-init.php#L14), so we're safe. 

As for the whitelist, that's meant to be temporary, but also can't be moved without moving the new plugin into a subdirectory. I'd prefer to do that only if I also break the plugin down into smaller pieces as you've suggested; I'm still pondering that.